dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -72,13 +72,13 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
                 changeDockerfiles(ns, pathToDockerfilesInParentRepo, imagesFoundInParentRepo, imageToTagMap, currUserRepo,
                         skippedRepos);
             } catch (IOException e) {
+                log.error(String.format("Error changing Dockerfile for %s", currUserRepo.getName()), e);
                 exceptions.add(e);
             }
         }
 
         if (!exceptions.isEmpty()) {
-            log.info("There were {} errors with changing Dockerfiles.", exceptions.size());
-            throw exceptions.get(0);
+            throw new IOException(String.format("There were %s errors with changing Dockerfiles.", exceptions.size()));
```

Comment:
I'm not stuck on this approach but it seemed odd to log the message and then just throw the first exception.